packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -73,35 +92,35 @@ const CompleteResetPassword = ({
     });
 
   const alertSuccessAndNavigate = useCallback(() => {
-    const successCompletePwdReset = ftlMsgResolver.getMsg(
-      'complete-reset-password-success-alert',
-      'Password set'
-    );
-    alertBar.success(successCompletePwdReset);
     navigate('/reset_password_verified', { replace: true });
-  }, [alertBar, ftlMsgResolver, navigate]);
+  }, []);
 
   const onSubmit = useCallback(
     async ({ newPassword }: FormData) => {
       try {
-        // TODO: completeAccountPasswordReset
-        // logViewEvent('verification.success');
+        await account.completeResetPassword(token!, code!, email!, newPassword);
         alertSuccessAndNavigate();
       } catch (e) {
-        // if token expired, re-render and show LinkExpired
-        // TODO metrics event for error
-        const errorCompletePwdReset = ftlMsgResolver.getMsg(
-          'complete-reset-password-error-alert',
-          'Sorry, there was a problem setting your password'
-        );
-        alertBar.error(errorCompletePwdReset);
+        setErrorCompletePwdReset(true);
       }
     },
-    [ftlMsgResolver, alertSuccessAndNavigate, alertBar]
+    [token, code, email]
   );
 
   return (
-    <>
+    <AppLayout>
+      {errorCompletePwdReset && (
+        <Banner
```

Comment:
I opted to use a Banner here instead of the alertBar, it looked a little better idk 😅

Reply:
Heh I think that's what we should use to better match to parity anyways.